package.json

```
@@ -7,7 +7,8 @@
     "build": "react-scripts build",
     "spj": "npx sort-package-json@1.48 package.json",
     "dev": "react-scripts start",
-    "stylelint": "stylelint 'src/*.{js,jsx}'"
+    "stylelint": "stylelint 'src/*.{js,jsx}'",
+    "dev-server": "nodemon server.js"
```

missing server script 
```suggestion
    "dev-server": "nodemon server.js",
    "server": "node server.js"
```

---

danke :)